src/components/FormComponent/_Select.tsx

```diff
@@ -0,0 +1,31 @@
+import MenuItem from '@mui/material/MenuItem';
+import TextField, { TextFieldProps } from '@mui/material/TextField';
+import { ChangeEvent, useState } from 'react';
+
+export type _SelectProps = Omit<TextFieldProps, 'select' | 'type'> & {
+  options?: { label: string; value: string }[];
+};
+
+export const _Select = (props: _SelectProps) => {
+  const [value, setValue] = useState('');
+
+  const onChange = (e: ChangeEvent<HTMLInputElement>) => setValue(e.target.value);
+
+  return (
+    <TextField
+      fullWidth
+      size="small"
+      data-testid={props.id}
+      label={props.label}
+      value={value}
+      onChange={onChange}
+      {...props}
+      select>
```

Comment:
Take a look to the console, it seems raise error
 > Received `true` for a non-boolean attribute `select`

Reply:
Seems fixed with the latest commit